qdrant-landing/content/documentation/frameworks/testcontainers.md

```diff
@@ -0,0 +1,45 @@
+---
+title: TestContainers
+weight: 2700
+---
+
+# TestContainers
+
+[TestContainers](https://testcontainers.com/) is an open source framework for providing throwaway, lightweight instances of databases, message brokers, web browsers, or just about anything that can run in a Docker container.
+
+Qdrant is available as a [TestContainers module](https://testcontainers.com/modules/qdrant/) in multiple languages facilitating the spawning of a Qdrant instance for end-to-end testing.
```

Comment:
The link goes to a Testcontainers doc which specifies `v1.7.4`. That conflicts with v1.8.1 in several places, in this file.

Could we specify `latest` in both docs?

Reply:
https://github.com/qdrant/landing_page/pull/710/commits/8381fb3f31b36f0a49bf5de1187061d001ffb4d2